packages/preload/src/index.ts

```
@@ -1457,6 +1457,13 @@ export function initExposure(): void {
     },
   );
 
+  contextBridge.exposeInMainWorld(
+    'createHash',
+    async (input: string, algorithm: string = 'sha512'): Promise<string> => {
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)